source/common/config/well_known_names.cc

```diff
@@ -235,6 +235,9 @@ TagNameValues::TagNameValues() {
   // Leaving: proxy_proto.(found|disallowed|error)
   addRe2(PROXY_PROTOCOL_VERSION,
          R"(^proxy_proto\.((?:<TAG_VALUE>\.)?versions\.v(<PROXY_PROTOCOL_VERSION>)\.)\w+$)");
+
+  // grpc.(<stat_prefix>).**
```

Comment:
I don't think it's the stat_prefix, but the cluster name, no?

Reply:
Just clarifying this, so that is indeed the **stat prefix**. This stat is only available through the [Google GRPC](https://www.envoyproxy.io/docs/envoy/v1.32.0/api-v3/config/core/v3/grpc_service.proto#envoy-v3-api-msg-config-core-v3-grpcservice-googlegrpc) client, which has mandatory string field for that prefix.